examples/data-objects/clicker-react/clicker-context/src/index.tsx

```diff
@@ -113,10 +114,11 @@ export class ClickerContext extends SyncedDataObject {
 }
 
 // ----- FACTORY SETUP -----
-export const ClickerContextInstantiationFactory = new DataObjectFactory(
-    "clicker-context",
-    ClickerContext,
-    [],
-    {},
-);
+export const ClickerContextInstantiationFactory =
+    new DataObjectFactory<ClickerContext, unknown, unknown, IEvent>(
```

Comment:
here too and looks like a lot of other places use `unknown, unknown` instead of `object, undefined`.  is it intentional?  (it's definitely a pain to introduce and suppress one error to fix another...)

Reply:
I've left these as typed by intellisense, but I filed #5489 to get a definitive answer on whether `unknown, unknown` is safe(st).